Convex_hull_3/examples/Convex_hull_3/quickhull_OM_3.cpp

```diff
@@ -32,8 +32,7 @@ int main(int argc, char* argv[])
   Mesh sm;
   
   CGAL::convex_hull_3(points.begin(), points.end(), sm);
-
-  CGAL::write_off((argc>2)?argv[2]:"out.off", sm);
```

Comment:
I think the generic `write_off` has been introduced in CGAL-4.11. And the PR #2275 has been tested in `integration` only, the branch for CGAL-4.11.